docs/operations/tiered-storage.md

```diff
@@ -89,9 +89,13 @@ After build successfully, there should be a `kafka-storage-x.x.x-test.jar` file
     # Note, please make sure the brokers need to have access to this directory
     rsm.config.dir=/tmp/kafka-remote-storage
     
-    # This needs to be changed if number of brokers in the cluster is more than 1
+    # For single broker cluster, set this to 1. Default is 3 for clusters with 3 or more brokers.
     rlmm.config.remote.log.metadata.topic.replication.factor=1
     
+    # The minimum number of replicas that must acknowledge a write to remote log metadata topic.
```

Comment:
```suggestion
    # The minimum number of insync replicas that must acknowledge a write to remote log metadata topic.
```

Reply:
https://github.com/apache/kafka/pull/20811#discussion_r2649487221 the same reply for this. thanks